cmd/harbor/root/login.go

```diff
@@ -30,6 +33,16 @@ func LoginCommand() *cobra.Command {
 				serverAddress = args[0]
 			}
 
+			if passwordStdin {
+				fmt.Print("Password: ")
```

Comment:
```suggestion
				fmt.Print("Enter Password: ")
```

This might be better.